README.md

```diff
@@ -1,32 +1,73 @@
-# Applicative option parser
-
-This package contains utilities and combinators to define command line option
-parsers.
+# optparse-applicative
 
 [![Continuous Integration status][status-png]][status]
 [![Hackage page (downloads and API reference)][hackage-png]][hackage]
 
+optparse-applicative is a haskell library for parsing options on the command line.
+
+Providing a powerful [applicative] interface to compose command line
+options and arguments, optparse-applicative provides all that is required for
+precise and concise CLI specifications for both simple and complex programs.
```

Comment:
I'd split this into two sentences, as it's too complex as is. Or even just make it a simpler introduction.
eg "optparse-applicative provides everything required ... "